Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py

```diff
@@ -1,6 +1,1031 @@
+             "noscript", "object", "ol", "optgroup", "option", "p", "param", "pre", "q", "samp", "script", "section",
+             "select", "small", "span", "strong", "style", "sub", "sup", "table", "tbody", "td", "textarea", "tfoot",
+             "th", "thead", "title", "tr", "tt", "ul", "var"]
+HTML_TAGS = set(HTML_TAGS)
+
+'''
+Define known shortened and drive domains
+'''
+
+DRIVE_URL_KEYWORDS = ['drive', 'transfer', 'formplus', 'dropbox', 'sendspace', 'onedrive', 'box', 'pcloud', 'icloud',
+                      'mega', 'spideroak', 'sharepoint']
+SHORTENED_DOMAINS = set(
+    ["adf.ly", "t.co", "goo.gl", "adbooth.net", "adfoc.us", "bc.vc", "bit.ly", "j.gs", "seomafia.net", "adlock.in",
+     "adbooth.com", "cutt.us", "is.gd", "cf.ly", "ity.im", "tiny.cc", "adfa.st", "budurl.com", "soo.gd",
+     "prettylinkpro.com", "shrinkonce.com", "ad7.biz", "2tag.nl", "1o2.ir", "hotshorturl.com", "onelink.ir", "dai3.net",
+     "9en.us", "kaaf.com", "rlu.ru", "awe.sm", "4ks.net", "s2r.co", "4u2bn.com", "multiurl.com", "tab.bz", "dstats.net",
+     "iiiii.in", "nicbit.com", "l1nks.org", "at5.us", "bizz.cc", "fur.ly", "clicky.me", "magiclinker.com",
+     "miniurl.com", "bit.do", "adurl.biz", "omani.ac", "1y.lt", "1click.im", "1dl.us", "4zip.in", "ad4.us", "adfro.gs",
+     "adnld.com", "adshor.tk", "adspl.us", "adzip.us", "articleshrine.com", "asso.in", "b2s.me", "bih.cc", "biturl.net",
+     "buraga.org", "cc.cr", "cf6.co", "dollarfalls.info", "domainonair.com", "gooplu.com", "hide4.me", "ik.my",
+     "ilikear.ch", "infovak.com", "itz.bz", "jetzt-hier-klicken.de", "kly.so", "lst.bz", "mrte.ch",
+     "multiurlscript.com", "nowlinks.net", "nsyed.com", "ooze.us", "ozn.st", "scriptzon.com", "short2.in",
+     "shortxlink.com", "shr.tn", "shrt.in", "sitereview.me", "sk.gy", "snpurl.biz", "socialcampaign.com", "swyze.com",
+     "theminiurl.com", "tinylord.com", "tinyurl.ms", "tip.pe", "ty.by"])
+
+
+def find_label_fields_candidates(incidents_df):
+    candidates = [col for col in list(incidents_df) if
+                  sum(isinstance(x, str) for x in incidents_df[col]) > 0.5 * len(incidents_df)]
+    candidates = [col for col in candidates if col not in LABEL_FIELDS_BLACKLIST]
+
+    candidate_to_values = {col: incidents_df[col].unique() for col in candidates}
+    candidate_to_values = {col: values for col, values in candidate_to_values.items() if
+                           all(isinstance(v, str) for v in values)}
+
+    # filter columns by unique values count
+    candidate_to_values = {col: values for col, values in candidate_to_values.items() if 0 < len(values) < 15}
+    candidate_to_values = {col: [v.lower() for v in values] for col, values in candidate_to_values.items()}
+    candidate_to_score = {col: 0 for col in candidate_to_values}
+    for col, values in candidate_to_values.items():
+        for v in values:
+            if any(w in v for w in LABEL_VALUES_KEYWORDS):
```

Comment:
consider to check also the field name, boolean
if the field contains:  "classifi", "type", "resolution",  "reason", "category", "disposition", "severity", "malicious", "tag" add more fields